app/modules/onboardin/api/user.api.ts

```diff
@@ -0,0 +1,18 @@
+import { AxiosPromise } from "axios";
+import { User } from "../models/User";
+
+export interface IUsersRequestParams {
+    sort?: string,
+    limit?: number,
+    page?: number,
+}
+
+export interface IUserApi {
+    getUsers(params?: IUsersRequestParams): AxiosPromise<User[]>,
+
+    getUserById(id: number): AxiosPromise<User>,
+
+    updateUser(user: User): AxiosPromise<User>,
```

Comment:
From the side of the backend would be better to add the id to the update method.